### DISCUSSION TITLE
Method to delete interaction messages

### DISCUSSION THREAD

**Discussion by Army-py** [kiki7000/discord.py-components#97] [Ideas]
Hello, 

I would have thought that you could add a method to delete messages from interactions sent with `interaction.send()` . This would be very useful for page systems for example to avoid having an interaction error.

**Comment by Damego** [comment]
`message = await interaction.send(..., ephemeral=False)
await message.delete()`
Also you cannot delete ephemeral message.

**Reply by Army-py** [reply]
Oh yes indeed I hadn't thought of that 
Thank you very much
